Update wireless.md

Link02211986 opened the pull request:
This update refreshes the Wireless and Basic Troubleshooting article to match the latest System76 support documentation. It has simplified commands making it easier and faster for users to resolve Wi-Fi and network issues.

I kept many of the original details while adding some of my own insights. Your feedback and thoughts on my updates would be greatly appreciated!

Commit:
Move article to archived

Commit:
Set the article to hidden

jacobgkau committed:
Improve wording & address feedback about home dir formatting

thomas-zimmerman committed:
Merge pull request #1138 from system76/promote-recovery

Work on Issue 971

Commit:
Merge branch 'issue-1150' of github.com:system76/docs into issue-1150

ahoneybun committed:
Merge branch 'master' into issue-1150

Commit:
Fix build issue

Commit:
Merge branch 'issue-1150' of github.com:system76/docs into issue-1150

Commit:
Fix lint issue

thomas-zimmerman committed:
Merge pull request #1157 from system76/issue-1150

Move roadmap article to archived

leviport committed:
Add note about new 90% default start threshold (#1163)

burden committed:
Update footer (#1162)

Socialhappiness committed:
adding images for new support article about elarging desktop

Socialhappiness committed:
add images for desktop zoom article

ahoneybun committed:
Add links to pang13

ahoneybun committed:
Work on adding the advanced luks article to the password article to combine them fixing issue 1161

ahoneybun committed:
address issue 1164

ahoneybun committed:
Update guides.md

thomas-zimmerman committed:
Merge pull request #1169 from system76/add-pang13

Add link to pang13

ahoneybun committed:
Merge branch 'master' into pang12-flex

Commit:
Fix lint issue

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update Other OS steps based on feedback (#1154)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update package manager (#1137)

* Update content

* Add commands

* Fix lint issue

* Update wording to directly mention files that will be created

* Pop!_Shop > Pop Shop
Update to not mention Pop Shop but Ubuntu Store -- Software & Updates

Update both for Apt > APT

* Fix spelling and formatting

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <thomas@system76.com>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

burden committed:
update @system76/components package (#1170)

* update @system76/components package

* update link

* spelled right

jonknapp committed:
Update wireless.md (#1143)

Add info about disabling Windows "Fast Startup" which was the cause of my missing wifi card after a driver update from the Windows side.

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Update driver software (#1127)

* Update order and switch to git versions

* Add io package

* Fix order

* Update Fedora steps

* Add content and reorder it

* Update system76-software.md

extended update around gnome-extension that it does not work on newer GNOME releases.

* Remove Paru line in Fedora steps

* Address feedback

* Update folder names

* Replace system76-firmware-daemon with base system76-firmware package

* Update repo

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update customize gnome (#1135)

* Update Pop!_OS spelling and add notes

* Remove Gnomenu as it does not support GNOME 42

* Remove Extension Update Notifier as it is no longer supported and it is built into extensions.gnome.org now

* Fix lint issue

* Address feedback

* Update customize-gnome.md

extend note on Dash-to-Dock and Cosmic-Dock

* Address feedback

* Fix Pop spelling

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

burden committed:
fix builds for node 18+ (#1174)

* support for node 18

* update @system76/design

Commit:
Update image

ahoneybun committed:
Merge branch 'master' into pang12-flex

thomas-zimmerman committed:
Merge pull request #1167 from system76/pang12-flex

address issue 1164

ahoneybun committed:
Add section for tech docs and guides (#1145)

* Add section for tech docs and guides

* Add keywords for discovery

* Add section in article to guides

* Remove section

* Remove keywords from incorrect article

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
New Fan Noise article (#1171)

* Init work on Fan Noise article

* Fix lint issues

* Fix lint issues

* Update fan-noise.md

Worked on the language

* Update fan-noise.md

...and additional language updates

* add image and fix lint issue

* Address feedback

* Update list for newer and older models

* Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update linux gaming (#1175)

* Update image to new Steam UI

* Update search screenshot and activities to application for 22.04

* Update screenshots

* Update wording

* Move using steam section

* remove note about GameHub as it works now.

* Update linux-gaming.md

fix feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

thomas-zimmerman committed:
Update password.md

updates to the LUKS commands.

Commit:
Fix lint issue

ahoneybun committed:
Merge branch 'master' into update-passwords

thomas-zimmerman committed:
Add note to use launch cables for firmware updates (#1177)

* Add note to use launch cables for firmware updates

Update launch_2-firmware-update.md

* Address feedback

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Merge branch 'master' into update-passwords

ahoneybun committed:
Update dock article (#1178)

* Update images and renaming article

* Remove Ubuntu 16.04 section as it is no longer supported

* Add back Dash to Dock

* Add content

* Update images and remove old ones

* Rename article to fix link

* Fix lint issue

* Fix lint issue and address feedback

* Remove Dash to Dock installation and instead use dconf editor

* Correct position of the default dock for Ubuntu and Pop!_OS and adjust other content

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <thomas@system76.com>

thomas-zimmerman committed:
Log in from live disk (#1180)

* Update login-from-live-disk.md

Added context to encryption section

* Add some additional comments on what the chroot commands are doing and fixing lint issues

* Address feedback

---------

Co-authored-by: Sara Mildenstein <115432964+sararocks@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Fix shortcuts and address feedback

ahoneybun committed:
Change table to command to check hardware (#1182)

burden committed:
remove gear up promo (#1188)

* remove gear up promo

* update omnibar

Spongecade committed:
Update Minecraft wiki link to new domain (#1187)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

mapachurro committed:
Remove duplicate instruction (#1189)

* Remove duplicate instruction

The instruction regarding taking a picture or screenshot of the instructions showed up twice. I left the later one, as I feel it provided better emphasis to an important point.

* Update system-firmware.md

* Update system-firmware.md

* Update system-firmware.md

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add Boot Menu from the thelio-r3 board (#1183)

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Add meer7 and darp9 (#1185)

* Add meer7

* Add darp9, meer7 and fix lint issue

---------

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Update screenshots and content for 22.04 (#1181)

* Update screenshots and content for 22.04

* Remove Beta line and fix grammar

---------

Co-authored-by: Francis Smith <fs1995@ymail.com>

ahoneybun committed:
Add note about the pang12/13 switch (#1179)

* Add note about the pang12/13 switch

* webcam: Reword article to better incorporate pang12/13 switch

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Francis Smith <fs1995@ymail.com>

Socialhappiness committed:
Create Desktop_zoom.md (#1165)

* Create Desktop_zoom.md

adding an article specifically for making desktop bigger

* work on fixing lint issues

* Rename file

* More lint fixes

* Fix lint issues

* Fix numbered list issue

* Another possible lint fix

* Update desktop-zoom.md

* Add lint rule to keep numbered list with images working

* Update desktop-zoom.md

* Add images

* Fix image links

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Update ubuntu articles (#1176)

* Add note about GNOME Shell extension for newer then Ubuntu 22.04

* Link to boot menu article for up to date steps and address feedback

* Address feedback about note

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Francis Smith <fs1995@ymail.com>

JoseJacsan committed:
Added Shotcut video editor description in list (#1191)

* Added Shotcut video editor description in list

* Added spelling error correction

* Add additional descriptors from the project's homepage

* Update media-production-on-pop.md

Fix lint issue

* fix (lint): Remove double line break at end of file

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

adamdicarlo0 committed:
Fix misplaced word and use "back up" (verb) instead of "backup" (#1193)

burden committed:
Update omnibar with link to Tuxmas page (#1194)

burden committed:
tuxmas sale week 2, update components lib + omnibar copy (#1195)

burden committed:
tuxmas sale week 3 (#1196)

garrettjwilke committed:
bash basics tutorial (#1197)

* bash basics tutorial

* Address feedback

* Address feedback

* Rewrite adding functionality section

---------

Co-authored-by: Garrett Wilke <garrett@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: leviport <levi@system76.com>

ahoneybun committed:
Update images and comment out CLI option as does not support that in v4 yet

Fix lint

thomas-zimmerman committed:
Merge pull request #1200 from system76/update-protonvpn-killswitch

Update images and comment out CLI option

thomas-zimmerman committed:
Merge branch 'master' into update-passwords

thomas-zimmerman committed:
Merge pull request #1166 from system76/update-passwords

Work on adding the advanced luks article to the password article

memachado committed:
Update desktop-environment.md (#1190)

* Update desktop-environment.md

This addition offers users a more comprehensive solution for removing KDE and its associated packages. This enhancement aims to provide clearer and more user-friendly instructions for managing desktop environments on Pop!_OS.

* Split regex from normal autoremove and add explanation

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>

macifell committed:
tuxmas phase 4

macifell committed:
Merge pull request #1205 from system76/tuxmas4

tuxmas phase 4

ahoneybun committed:
Add meer8 to guide and windows articles

bbarker committed:
Update difference-between-pop-ubuntu.md (#1209)

Fix typo in keyboard shortcut for switching workspaces

macifell committed:
Remove tuxmas

macifell committed:
Merge branch 'master' into remove-tuxmas

macifell committed:
Merge pull request #1211 from system76/remove-tuxmas

Remove tuxmas

seansawyer committed:
Update guides.md to include Lemur Pro 12 (lemp12) tech doc (#1212)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

PeterGrand committed:
Use recursive binding before chroot (#1204)

By using -R instead of -B, we ensure that the nested mount of /sys/firmware/efi/efivars/ becomes mounted in our target filesystem, which allows us to do various boot repairs with an EFI system.

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

samcodee committed:
Update article about Differences between Pop!_OS and Ubuntu (#1203)

* Add updated intel nvidia

* Fix typo: "Backported"

* Add new image for download screen

also added info about rpi 4

* Rename intel_nvidia.png to intel-nvidia-rpi.png

* Delete old image

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Add pang13 and pang14 to missing spots in articles (#1213)

ahoneybun committed:
Add nix-shell file to use nix for testing

ahoneybun committed:
Update shell.nix

remove shellHooks

ahoneybun committed:
add launch_3 and launch_heavy_3 tech doc links

jklgrasso committed:
Update hardware-failure.md

Changed the gpu-burn example to use tc instead of d to create more of a workload when testing.

jklgrasso committed:
Update hardware-failure.md

Adjusted for RTX GPUs to use -tc and removed the -d switch for GTX GPUS.

thomas-zimmerman committed:
Merge pull request #1221 from system76/gpu-burn-d-to-tc

Update hardware-failure.md

ahoneybun committed:
Fix typo for Packages

thomas-zimmerman committed:
Merge pull request #1207 from system76/add-meer8

Add meer8

thomas-zimmerman committed:
Merge branch 'master' into add-nix-shell

thomas-zimmerman committed:
Merge branch 'master' into update-launch

thomas-zimmerman committed:
Merge pull request #1219 from system76/update-launch

add launch_3 and launch_heavy_3 tech doc links

thomas-zimmerman committed:
Merge branch 'master' into add-nix-shell

thomas-zimmerman committed:
Merge pull request #1216 from system76/add-nix-shell

Add nix-shell file to use nix for testing

ahoneybun committed:
Clean up and remove old install Ubuntu images for install-ubuntu article

ahoneybun committed:
Update/Move content and update screenshots

ahoneybun committed:
Add more images

ahoneybun committed:
Add Software Updater screenshots

ahoneybun committed:
Remove old screenshots

ahoneybun committed:
update: clean up the windows table and add missing systems (#1214)

* update: clean up the windows table and add missing systems

* fix: typo and lint issue

* Update windows.md

Remove left over from merge conflict

---------

Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

ahoneybun committed:
Update deploy.yml (#1225)

* Update deploy.yml

* Update ci.yml

* Update deploy.yml

macifell committed:
Revert "Update deploy.yml (#1225)"

This reverts commit 5427233722b0a763646de0ca615ef10c81c59432.

thomas-zimmerman committed:
Merge pull request #1227 from system76/revert-1225-update-workflow

Revert "Update deploy.yml"

ahoneybun committed:
Finish up article with initial setup screenshots

leviport committed:
Merge branch 'master' into update-ubuntu-24.04

ahoneybun committed:
Add images for 23.10 to 24.04 upgrade

leviport committed:
Merge branch 'master' into update-upgrade-ubuntu

ahoneybun committed:
Update Ubuntu supported release and links for making USB while removing DVD reference

ahoneybun committed:
Address the rest of the feedback minus two from the review

ahoneybun committed:
Update GRUB and LUKS/Plymouth screenshots

ahoneybun committed:
Cleaned up remaining fixes in the review

ahoneybun committed:
Updates the wording about third-party software and media support selection in the installer

ahoneybun committed:
Address feedback

jacobgkau committed:
fix: Grammar & typos

jacobgkau committed:
fix: Additional ordering, grammar, and consistency tweaks

thomas-zimmerman committed:
Merge pull request #1228 from system76/update-ubuntu-24.04

Update install-ubuntu article for 24.04

ryanbaekr committed:
Add gnome-shell-extensions to the packages needed for Ubuntu (#1232)

* Add gnome-shell-extensions to the packages needed for Ubuntu

Ubuntu 24.04 doesn't ship with gnome-shell-extensions

* Update pop-shell.md

* improv: List reason for installing extension-prefs package

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

taivlam committed:
Fix typo in launch-keyboard.md (#1234)

Use "its" (possessive), instead of "it's" (contraction)

taivlam committed:
Fix typo in launch_2-firmware-update.md (#1233)

Remove extra "for"

Co-authored-by: Levi Portenier <levi@system76.com>

jacobgkau committed:
Add ROCm article (#1231)

ahoneybun committed:
Merge branch 'master' into fix-typo

Commit:
Merge pull request #1223 from system76/fix-typo

Fix typo for Packages

ahoneybun committed:
Update model list and remove unsupported OSes (#1224)

ahoneybun committed:
Redo bootloader (#1198)

* Reorder article and update sections

* Update notes about BIOS installs

* Fix header for encrypted disk

* Add Pop!_OS GRUB steps and fix lint issue

* Address feedback

* Update bootloader.md

Removed > on line 85 to (hopefully) fix markdown failure.

---------

Co-authored-by: brandonbaez7 <160044194+brandonbaez7@users.noreply.github.com>

trenner1 committed:
Update pop-keyboard-shortcuts.md

add the key strokes for moving active windows between upper and lower displays

jacobgkau committed:
Add recent tech docs models & link to tech docs homepage (#1237)

thomas-zimmerman committed:
add note for EC update (#1239)

* add note for EC update

Firmware updates that include EC firmware updates require a full power cycle to complete. Added a note suggesting either AC power unplug/plug or holding power button for 15 seconds.

* Update system-firmware.md

..and fix language.

Commit:
Update upgrade-ubuntu.md

Removed documentation referring to DVDs. Changed documentation referring to the Activities Overview, which now refers to the Applications menu and the Show Apps button. Highlighted that the Software Updater may take a moment to show the settings page.

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

ahoneybun committed:
Update upgrade-ubuntu.md

fix: lint issue

spaceoden committed:
Add note about need to disable ppd on NixOS (#1240)

* Add Nixos power note to system76-software.md

ppd (enabled by default on Gnome NixOS installs) must be disabled for system76-power to start. Include that info in docs.

* Update system76-software.md

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

ws909 committed:
Update the steps to generate a log file in freezing-behavior.md (#1241)

* Update the steps to generate a log file in freezing-behavior.md

The existing explanation was for a previous version of the system.

* Update freezing-behavior.md

Readd the log file creation in the System76 Driver in addition to in the Settings application

* Update freezing-behavior.md

fix Pop!_OS

* Update freezing-behavior.md

* Update freezing-behavior.md

fix lint

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

UrielCh committed:
Update cuda.md, replace cuda 12.2 with 12.4 (#1238)

* Update cuda.md, replace cuda 12.2 by 12.5

Cuda 12.2 is deprecated, and display nasty warning.

the driver version 555 is currently 6 day old.

the driver version 550 was built on top of cuda 12.4.

* replace cuda 12.5 by 12.4

---------

Co-authored-by: thomas-zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

jessecambon committed:
Add info on Docker usage to ROCm install instructions (#1242)

* Added info on docker usage to rocm.md

See comments here for more information https://gist.github.com/guilt/6c901f7ac0a726685b6334798da77c00?permalink_comment_id=5131866#gistcomment-5131866

* adjusted docker terminology in rocm.md

* Add docker site link to rocm.md

* Minor grammar corrections

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

itsfaizankhan committed:
Fixed typo in switch-from-macos-to-popos.md

jacobgkau committed:
improv (pop-key-shortcuts): Improve wording & symbol consistency

esplinr committed:
Update system-firmware.md (#1243)

* Update system-firmware.md

Make clear what happens if the firmware updater is run with external devices plugged in.

* Update system-firmware.md

Incorporating feedback

* enhance (firmware): Split additional info into sub-bullets

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>

jacobgkau committed:
bluetooth: Add btusb reload commands (from #1217) (#1246)

jacobgkau committed:
bootloader: Make bind mounts recursive for EFI vars support

nonergodic committed:
Add bullet point to Before You Begin section in system-firmware doc (#1134)

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>

jacobgkau committed:
Codecs update (#1247)

* Install libdvd-pkg

* codecs: Make command instructions consistent & include TUI prompt info

---------

Co-authored-by: mrvictory1 <108172910+mrvictory1@users.noreply.github.com>
Co-authored-by: Levi Portenier <levi@system76.com>

burden committed:
Update banner image (#1249)

ahoneybun committed:
Fix lint

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

ahoneybun committed:
Update upgrade-ubuntu.md

fix lint

jacobgkau committed:
fix: Minor grammar/typo corrections

jackpot51 committed:
Do not keep gh-pages history

Keeping the history of gh-pages adds a lot of bloat when trying to clone the repository. The force_orphan option added pushes just one commit with no prior history to the gh-pages branch on every change, which should significantly reduce the size of the repository.

ahoneybun committed:
Merge branch 'master' into update-upgrade-ubuntu

thomas-zimmerman committed:
Merge pull request #1229 from system76/update-upgrade-ubuntu

Update upgrade-ubuntu article for 24.04

jacobgkau committed:
List Pop!_Shell branches for various Ubuntu versions (#1252)

* shell: List branches for different Ubuntu versions

* shell: Other minor grammar fixes

* shell: Mark code block type

jacobgkau committed:
Add article: VirtualBox Black Screen on Threadripper (#1256)

Nested paging must be disabled on some Threadripper and Threadripper Pro
products in order for VirtualBox guests to display properly.

I compressed the larger screenshots with WebP, but left the smaller ones
as optimized PNG because the compression looks bad when they're scaled up.

ahoneybun committed:
fix dmidecode typo (#1254)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Levi Portenier <levi@system76.com>

ahoneybun committed:
Update ubuntu-basics article for 24.04 (#1230)

* Remove old 16.04 screenshots

* Add Ubuntu 24.04 screenshots

* Fix typo and add more screenshots

* Remove older 21.04 and below iamges and update 24.04 images

* Update wording about adding to the launcher

* Ubuntu Basics: Minor style tweaks

---------

Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Levi Portenier <levi@system76.com>

mlundyCO committed:
Update system-firmware.md (#1251)

Moved sentences about EFI to appropriate section.

Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Aaron Honeycutt <aaron@system76.com>

basujindal committed:
Update graphics-switch-ubuntu.md (#1222)

* Update graphics-switch-ubuntu.md

Added note.

* Update graphics-switch-ubuntu.md

Add another note

---------

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@protonmail.com>
Co-authored-by: Levi Portenier <levi@system76.com>

ahoneybun committed:
Add photos transfer article (#1210)

* Init work on new article

* Continue work on article adding more sections and images

* Work on wording

* Add Pixel/Android steps and more images/text

* address feedback

* update text

* Verbiage tweaks in photo transfer article.

---------

Co-authored-by: Levi Portenier <levi@system76.com>
Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>
Co-authored-by: Thomas Zimmerman <41023059+thomas-zimmerman@users.noreply.github.com>

burden committed:
update footer (#1258)

ahoneybun committed:
Remove accessories (#1259)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

ahoneybun committed:
Update windows.md

add addw4 to the table

Commit:
updates link to the guide article rather than per model and adds photo for the v2 board's power button

thomas-zimmerman committed:
Merge pull request #1266 from system76/update-power-on-failure

Updating the page to be more clear and include Thelio Io v2

Commit:
change note about the Pangolins to a table

Commit:
add pang15 to the list of models

Commit:
update table based on feedback

Commit:
address feedback

thomas-zimmerman committed:
Merge pull request #1268 from system76/update-battery-thresholds-article

Update battery thresholds article

leviport committed:
Merge branch 'master' into add-addw4-windows

thomas-zimmerman committed:
Merge pull request #1260 from system76/add-addw4-windows

Update windows.md

leviport committed:
Add meer9 to open firmware & EC list, and fix table whitespace (#1271)

* Add meer9 to open firmware & EC list, and fix table whitespace

* List meer9 EC as N/A (it has no EC, open or otherwise)

thomas-zimmerman committed:
Add a new support document as a landing page for URLs in old support … (#1264)

* Add a new support document as a landing page for URLs in old support messages to allow both an explnation
and quick way to open support ticket if they still need help.

* Address some lint issues with spaces.

* respond and update language from jacobgkau review

* update for AR language

* Address Mikes updates

* address points raised that github UI was hiding.

* Additional link & grammar tweaks

* Remove unnecessary footer & additional link tweak

* Additional link tweaks & punctuation fix

* Additional clarifications & formatting tweaks

---------

Co-authored-by: Jacob Kauffmann <jacobgkau@users.noreply.github.com>
Co-authored-by: Levi Portenier <levi@system76.com>

thomas-zimmerman committed:
Update system76-driver.md (#1274)

Remove the `apt update` after adding the PPA and add a `apt upgrade` afterwords. Adding the PPA will always get updated release files. Need the upgrade to upgrade packages in the PPA.

thomas-zimmerman committed:
Update install-ubuntu.md (#1275)

* Update install-ubuntu.md

Add a "apt upgrade" after the PPA is added.

* Update install-ubuntu.md

remove apt update after adding the PPA as it runs by default now.

---------

Co-authored-by: Aaron Honeycutt <aaron@system76.com>
Co-authored-by: Levi Portenier <levi@system76.com>

Commit:
move battery article as it is now incorrect for 22.04 and above

ahoneybun committed:
update action/cachev2 to v4 to fix build issue (#1278)

Co-authored-by: Aaron Honeycutt <aaronhoneycutt@proton.me>

leviport committed:
Clarify hibernation disclaimer sentence (#1273)

Co-authored-by: Aaron Honeycutt <aaron@system76.com>

ahoneybun committed:
Merge branch 'master' into fix-issue-1201

thomas-zimmerman committed:
Merge pull request #1277 from system76/fix-issue-1201

move battery article as it is now incorrect for 22.04 and above

raxod502 committed:
Add tips about getting gpu-burn to compile

jacobgkau committed:
Switch CUDA package & slight clarifications

thomas-zimmerman committed:
Merge pull request #1279 from raxod502/patch-1

Add tips about getting gpu-burn to compile

kspeakman committed:
note on etcher data collection

burden committed:
publish web infra page

jacobgkau committed:
fix: Put title in title case

thomas-zimmerman committed:
Merge pull request #1286 from system76/visible-infra-page

publish web infra page

thomas-zimmerman committed:
Update deploy.yml

Update actions/cache to v4 to fix deploy failures on github.
Should fix deploy error on [PR#1286](https://github.com/system76/docs/pull/1286)

AlexisSweetMcKay committed:
Update boot-menu.md

Added link to open firmware models page for convenience

AlexisSweetMcKay committed:
Update boot-menu.md

Amended ! error

AlexisSweetMcKay committed:
Update boot-menu.md

Changed link to open firmware models text in chart

EricMTodd committed:
Update boot-menu.md

Added a catch-all statement instructing users to try all the FX keys if the ones listed on the boot-splash or in this article don't work.

Changed BIOS to BIOS/UEFI.

EricMTodd committed:
Update boot-menu.md

Made grammatical and punctuation adjustments suggested by @jacobgkau

thomas-zimmerman committed:
Merge pull request #1289 from system76/EricMTodd-patch-1

Update boot-menu.md

thomas-zimmerman committed:
Update boot-menu.md

Fix the link.

thomas-zimmerman committed:
Merge branch 'master' into AlexisSweetMcKay-patch-1

thomas-zimmerman committed:
Merge pull request #1288 from system76/AlexisSweetMcKay-patch-1

Update boot-menu.md

wackbyte committed:
Add meer8, previous models, and meer10 to open firmware list

wackbyte committed:
Add meer9 to tech docs link list

wackbyte committed:
Fix formatting of "Pop!_OS" in article description

wackbyte committed:
Fix typo

wackbyte committed:
Re-add CC BY-SA 4.0 license (#1297)

wackbyte committed:
Update Luanti/Minetest article (#1295)

- Minetest rebranded as Luanti ([blog post](https://blog.luanti.org/2024/10/13/Introducing-Our-New-Name/)).
- It would be more accurate to refer to Luanti itself as an "engine" rather than a "game"--[their home page](https://www.luanti.org/) also calls it an "engine".
- It can now be [installed from a PPA](https://www.luanti.org/downloads/), simplifying the installation process greatly.
- Content is now available from [ContentDB](https://content.luanti.org/), which is accessible in-game, further simplifying the installation process.
- Multiplayer still works the same, so there was no need to update that section.

Co-authored-by: Levi Portenier <levi@system76.com>

kuchtact committed:
Fix incorrect usage of effect instead of affect (#1302)

EricMTodd committed:
Update flash-firmware-from-usb.md (#1303)

Fixed a small typo on line 65 that read "coping the file(s)" to "copying the file(s)".

Co-authored-by: Levi Portenier <levi@system76.com>

EricMTodd committed:
Update terminal-basics.md

Added a missing period after `ls` on line 47. Added additional information detailing `cd` by itself will return the user to the home directory.

EricMTodd committed:
Merge pull request #1305 from system76/EricMTodd-patch-1

Update terminal-basics.md

thomas-zimmerman committed:
Merge pull request #1292 from wackbyte/meer-open-firmware

Add meer8, previous models, and meer10 to open firmware list

jacobgkau committed:
chore(ci): Bump Nix Shell's Node.js from 18 to 20

thomas-zimmerman committed:
Merge pull request #1306 from system76/nix-node

chore(ci): Bump Nix Shell's Node.js from 18 to 20

jacobgkau committed:
Add new dependency to DaVinci Resolve installation

Also reorder dependencies to be in alphabetical order

thomas-zimmerman committed:
Merge pull request #1308 from system76/resolve-libglu

Add new dependency to DaVinci Resolve installation

WatchMkr committed:
Create pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Update pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Update pop-cosmic-keyboard-shortcuts.md

jacobgkau committed:
Add another dependency for DaVinci Resolve installation

WatchMkr committed:
Typo fix pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Add stacking shortcut pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Fix typo pop-cosmic-keyboard-shortcuts.md

WatchMkr committed:
Add missing ) pop-cosmic-keyboard-shortcuts.md

thomas-zimmerman committed:
Merge branch 'master' into resolve-wayland

jacobgkau committed:
Fix spacing lints in pop-cosmic-keyboard-shortcuts

thomas-zimmerman committed:
Merge pull request #1310 from system76/resolve-wayland

Add another dependency for DaVinci Resolve installation

GeorgeAverkin committed:
Update pop-cosmic-keyboard-shortcuts.md (#1311)

Fix "Switch between added language/input source" shortcut

jacobgkau committed:
Document oryp5's Open Firmware transition

thomas-zimmerman committed:
Merge pull request #1313 from system76/oryp5-transition

Document oryp5's Open Firmware transition

jacobgkau committed:
Homepage tweaks

thomas-zimmerman committed:
Merge pull request #1314 from system76/updates

Homepage tweaks

Socialhappiness committed:
Update fan-noise.md

Adding Mechanical Troubleshooting Section to put more fan noise troubleshooting into one place

Socialhappiness committed:
Update fan-noise.md

corrected duplicate section

jacobgkau committed:
Correct header hierarchy & small punctuation tweaks

thomas-zimmerman committed:
Merge pull request #1318 from system76/Socialhappiness-patch-4

Update fan-noise.md

Socialhappiness committed:
Make article more general & add warning about losing logins (#1300)

Co-authored-by: Jacob Kauffmann <jacob@system76.com>
Co-authored-by: Levi Portenier <levi@system76.com>

thomas-zimmerman committed:
Update log-files.md

Get manual log creation closer to what pop-support does.

thomas-zimmerman committed:
Merge pull request #1328 from system76/thomas-zimmerman-patch-1

Update log-files.md

jacobgkau committed:
Clarify support tickets are only for customers

jacobgkau committed:
Remove note markers

Link02211986 committed:
Update wireless.md

Link02211986 committed:
Update wireless.md

Socialhappiness commented on content/wireless.md:
```diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
-WiFi issues are influenced by many different factors, including:
+If your computer can’t connect to wireless networks or the connection is unstable, use this guide to diagnose and resolve the issue.  
+Start with the quick checks, then follow the targeted diagnostic and recovery steps.  
+Include logs when you ask for support.
```
include logs sentence can be removed, we will add it at the bottom

Socialhappiness commented on content/wireless.md:
````diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
+  sudo apt remove backport-iwlwifi-dkms
+```
+Then restart your computer.
+Temporarily use a phone hotspot to confirm whether the network or your computer is the issue.
````
4. Temporarily use a phone hotspot to confirm whether the network or your computer is the issue.

Socialhappiness commented on content/wireless.md:
````diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
+```
+Then restart your computer.
+Temporarily use a phone hotspot to confirm whether the network or your computer is the issue.
+Boot from a Live USB of your distribution to determine whether the issue exists outside your installed system.
````
5. [Boot from a Live USB](https://support.system76.com/articles/live-disk) of your distribution to determine whether the issue exists outside your installed system.

Socialhappiness commented on content/wireless.md:
```diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
+Temporarily use a phone hotspot to confirm whether the network or your computer is the issue.
+Boot from a Live USB of your distribution to determine whether the issue exists outside your installed system.
+
+Router and Access Point Recommendations
```
## Router and Access Point Recommendations

Socialhappiness commented on content/wireless.md:
```diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
+
+journalctl -b | grep -i network — View network-related boot logs.
+
+dmesg | grep -i -E 'wifi|wlan|firmware|ieee80211|rtl|brcm|ath' — View driver and firmware kernel logs.
```
All of the commands in this section need formatted correctly. One line tells what the command does, and the next line is the formatted command. 
For example:
List all network interfaces and IP addresses confirms your Wi-Fi interface (usually wlp2s0 or wlan0) is recognized.
```bash
ip a
```

Socialhappiness commented on content/wireless.md:
```diff
@@ -207,3 +337,16 @@ Sometimes the newest version of the firmware will clear up occasional bugs.  Ple
 
 If you are dual booting Windows, you may lose access to your wifi card entirely after running driver/OS updates in Windows. You may be able to gain access to your wifi card again by disabling "Fast Startup" in the Windows power options before booting back into Pop!_OS.
+
+## When to Contact Support
```
## Contact System76 Support

If you purchased a System76 computer and you’ve tried all the steps above, but your wireless connection still isn’t working as expected, please collect the output from the diagnostic commands and contact [System76 Support.](https://system76.com/contact/support)

Socialhappiness reviewed (changes requested):
Thank you for adding your advice and reworking the article! I've suggested a few changes. Let me know if you have any questions.

Link02211986 committed:
Update wireless.md

All requested edits have been completed, and the document is now ready for review. Thank you for the helpful feedback I appreciate your time and guidance!

Socialhappiness reviewed (changes requested):
Needs the list formatting fixed and check consistency for using the word Wi-Fi versus WiFi or wifi since it is used differently throughout the article. Also, line 24 can be replaced with the suggested heading. Looks good!

Link02211986 committed:
Update wireless.md

All requested edits have been completed, and the document is now ready for review. Thank you for the helpful feedback I appreciate your time and guidance!

Link02211986 committed:
Update wireless.md

The requested changes have been made. I’ve fixed the list formatting, standardized the use of Wi-Fi throughout the article, and updated the heading to Initial Troubleshooting.

Apologies for the oversight, and thank you for the helpful review.

Link02211986 committed:
Update wireless.md

Fixed markdownlint issues by removing trailing punctuation from headings and trimming trailing whitespace to ensure consistent formatting and pass CI checks.

Link02211986 committed:
docs: fix lint issues in wireless.md

Link02211986 committed:
docs: fix lint issues in wireless.md

Link02211986 committed:
fix(CI/lint): remove trailing punctuation and whitespace

Link02211986 committed:
fix(CI/lint): remove trailing punctuation and whitespace

Link02211986 commented:
I’ve added my documents here for review. I’d appreciate any guidance on what needs to be addressed before this can be merged.

jacobgkau commented on content/wireless.md:
```diff
@@ -1,10 +1,10 @@
 ---
-title: Wireless Troubleshooting
+title: Wireless and Basic Troubleshooting
```
This new name doesn't make sense to me. "Basic Wireless Troubleshooting" would make more grammatical sense than "Wireless and Basic Troubleshooting." But does adding the word "basic" add any value or make this article easier for people to find?

jacobgkau commented on content/wireless.md:
```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+
+## Initial Troubleshooting
+
+1.Reboot the router/modem and the computer.
```
Please put a space after the `#.` in ordered lists. This is currently being rendered as a series of regular paragraphs instead of a list since it's not valid Markdown for a list.

jacobgkau commented on content/wireless.md:
```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+
+2.Toggle Airplane Mode by pressing **Fn + F11 →.**
+
+3.If your Wi-Fi issues started after an update, try removing the backported Wi-Fi driver using the command line. Open the Terminal by pressing Super+t and type the following and press Enter:
```
This suggestion needs to be more targeted. `backport-iwlwifi-dkms` is not installed by default, so it would only be present if someone installed it manually. Additionally, this package would only affect Intel wireless cards.

This is also a run-on sentence with multiple `and` connectors and no commas. If there are at least three steps, you need to start including commas instead (or otherwise rephrase).

jacobgkau commented on content/wireless.md:
````diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+3.If your Wi-Fi issues started after an update, try removing the backported Wi-Fi driver using the command line. Open the Terminal by pressing Super+t and type the following and press Enter:
+
+```bash
+  sudo apt remove backport-iwlwifi-dkms
````
Do not include spaces at the beginning of a command. The only thing that would do is prevent the command from showing up in the user's `~/.bash_history` file.

jacobgkau replied:
There are still two spaces at the beginning of this command.

jacobgkau commented on content/wireless.md:
```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+
+## Router and Access Point Recommendations
+
+Make sure your access point is configured for stability and compatibility.
```
This looks like an unordered list. If this is how you want to organize the section, then you should format it as a Markdown unordered list by starting each list item with `- ` or `* `.

jacobgkau commented on content/wireless.md:
```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+
+Deep MAC filtering
+
+Ensure your device’s MAC address isn’t being filtered
```
This sentence is also missing punctuation.

jacobgkau commented on content/wireless.md:
```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+
+## NetworkManager and Configuration Tips
+
+NetworkManager controls Wi-Fi connections on most Linux systems.
```
It might be safer to make this a more qualified statement like "NetworkManager controls Wi-Fi connections on many Linux systems, including on Pop!_OS and Ubuntu systems by default."

jacobgkau commented on content/wireless.md:
````diff
@@ -107,7 +261,7 @@ If you have trouble with a Bluetooth headset and keeping a steady downlink speed
+Another way to help with Wi-Fi issues is to turn off power management for the hardware. To do so, edit the configuration file with this command:
 
 ```bash
 sudo gedit /etc/NetworkManager/conf.d/default-wifi-powersave-on.conf
````
Diddo on the above regarding Gedit.

Link02211986 committed:
Fix wireless troubleshooting docs

jacobgkau commented on content/wireless.md:
```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
+
+If the device is detected but unstable, verify the driver and firmware setup.
+
+Confirm the kernel has loaded the correct driver and firmware.
```
Still relevant.

jacobgkau commented on content/wireless.md:
```diff
@@ -206,4 +348,8 @@ Sometimes the newest version of the firmware will clear up occasional bugs.  Ple
+
+## Contact System76 Support
+
+If you purchased a System76 computer and you’ve tried all the steps above, but your wireless connection still isn’t working as expected, please collect the output from the diagnostic commands and [contact support](https://system76.com/contact/support).
```
You didn't need to remove System76 from the link name. It should say `contact System76 support`.

jacobgkau reviewed (changes requested):
Thanks for making some of those changes. Some of the comments I made don't seem to have been addressed. I'm open to discussion if you disagree with or have questions with any of the points I made. (GitHub might hide some of the comments if there are too many-- you might have to click a "Load more" link or something like that to see all of them.)

Link02211986 committed:
Clean up wireless.md per review

Link02211986 committed:
Clean up wireless.md per review

jacobgkau committed:
Put package name in backticks for inline code formatting

jacobgkau committed:
Add the rest of the list items to the list

jacobgkau committed:
Add description back to command

jacobgkau committed:
Better phrasing for keyboard shortcut description

jacobgkau committed:
Remove obsolete CRDA info with non-existent config file

jacobgkau committed:
Put descriptions before commands, remove redundant commands...
...and other cleanup.

jacobgkau committed:
Move additional list sub-content under the list items
